pyproject.toml

```diff
@@ -236,7 +236,7 @@ exclude = '''
 '''
 
 [tool.pytest.ini_options]
-addopts = ["-rfEX", "-pno:warnings"]
+addopts = ["-rfEX", "-pno:warnings", "-pbentoml.testing.pytest.plugin"]
```

Comment:
Why did you disable warnings here again?

Reply:
oh to silent useless warning from tensorflow and flax.

Reply:
Which ones? Can we fix those without silencing warnings globally?

Reply:
Those warnings aren't related to us. It is a protobuf deprecation on Tensorflow end.